ddtrace/debugging/_probe/remoteconfig.py

```diff
@@ -288,13 +288,14 @@ class DebuggerRemoteConfigSubscriber(RemoteConfigSubscriber):
     """
 
     def __init__(self, data_connector, callback, name, status_logger):
-        super().__init__(data_connector, callback, name)
+        super().__init__(data_connector, None, name)
         self._configs: Dict[str, Dict[str, Probe]] = {}
         self._status_timestamp_sequence = count(
             time.time() + di_config.diagnostics_interval, di_config.diagnostics_interval
         )
         self._status_timestamp = next(self._status_timestamp_sequence)
         self._status_logger = status_logger
+        self._probe_callback = callback
```

Comment:
```suggestion
        self._rc_callback = callback
```

Reply:
This was called probe_callback because this class inherit from a remote config class that already defined a callback with another mechanism. That's why I did not want to give it a remote config related name.

Are you sure this is what you want?

Reply:
Ah I see, so there's the potential for a clash perhaps. Maybe `_debugger_callback` then? 🤔